src/main/java/com/sparta/popupstore/domain/review/service/ReviewService.java

```diff
@@ -17,10 +21,35 @@ public class ReviewService {
     private final ReviewRepository reviewRepository;
     private final PopupStoreRepository popupStoreRepository;
 
-    public ReviewCreateResponseDto createReview(User user,Long id , ReviewCreateRequestDto requestDto) {
-        PopupStore popupStore = popupStoreRepository.findById(id).orElseThrow();
+    public ReviewCreateResponseDto createReview(User user,Long popupStoreId , ReviewCreateRequestDto requestDto) {
+        PopupStore popupStore = popupStoreRepository.findById(popupStoreId)
+            .orElseThrow(() -> new EntityNotFoundException("해당 팝업스토어가 없습니다."));
         Review review = requestDto.toEntity(user, popupStore);
         reviewRepository.save(review);
         return new ReviewCreateResponseDto(review);
```

Comment:
```suggestion
        Review review = requestDto.toEntity(user, popupStore);
        return new ReviewCreateResponseDto(reviewRepository.save(review));
```
save한 review가 아니면 id가 null 입니다.